<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCallFwd</class>
 <widget class="QDialog" name="DlgCallFwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="cbCallFwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Forward Call</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GrpFwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>On No Answer Setting</string>
   </property>
   <widget class="QCheckBox" name="chkNoAnswer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>On No Answer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNoAnsVM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Voice Mail</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNoAnsExtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboNoAnsExtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GrpBusy">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>On Busy Setting</string>
   </property>
   <widget class="QCheckBox" name="chkOnBusy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>On Busy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbBusyOnVM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Voice Mail</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbBusyExtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBusyExtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
